platform/android/gradle/dependencies.gradle

```diff
@@ -9,6 +9,7 @@ ext {
 
     versions = [
             mapboxServices: '2.2.9',
+            mapboxTelemetryVersion: '3.0.0-SNAPSHOT',
```

Comment:
Is it necessary to rely on a SNAPSHOT? If so, upgrading to a stable version needs to be ticketed.